rest/resources/rest.json

```diff
@@ -5,6 +5,9 @@
   },
 
   "port": 3000,
+  "protocol": "HTTPS",
+  "sslKeyPath": "",
+  "sslCertificatePath": "",
```

Comment:
question: are we using a separate certificate for REST => client as opposed to the one already used to communicate with the server?

cc: @gimer

Reply:
Manually installed/managed REST applications(outside of symbol-bootstrap) will default to HTTPS, meaning the user has to have a valid DNS record and a valid SSL Certificate issued by a Trusted Authority, has to configure the key and cert in the rest.json and he will be maintaining his certificates and domain records himself. That's what we agreed on the Discord AFAIK.

Reply:
@Jaguar0625 y, it's a separate one